### PR TITLE
213 get mempool test proto

### PR DESCRIPTION
**Pull request by SherifSK**
This pull request includes updated `GetMempoolResponse` attribute name and type (MapField -> RepeatedField).

### CODE REVIEW

File: src/Rpc.proto [hunk 1]
```diff
@@ -204,7 +204,7 @@ message GetMempoolRequest {
 }
 
 message GetMempoolResponse {
-    map<string, string> info = 1;
+    repeated string mempool = 1;
```
Comment (nshCore) [inline src/Rpc.proto:207]: Not sure what's going on here? any details

Comment (SherifSK) [reply src/Rpc.proto:207]: I just simplified the data sent.  I found there is no use for the MapField which is basically a key value pair that included the an id (key) and the transaction (value). The list returned is sufficient for the response handler to function properly.

Comment (nshCore) [reply src/Rpc.proto:207]: Does this mean you're doing the formatting node side rather in the cli?

Comment (SherifSK) [reply src/Rpc.proto:207]: No. I was creating a list of key/value pairs.  Currently, I get the mempool content and return them all as is to the CLI to handle/format in the way the response handler on the CLI side see fit.

Comment (nshCore) [reply src/Rpc.proto:207]: Maybe @monsieurleberre is best to comment on this if he can.

**Pull request merged**